# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
float32 progress_time
float32 progress_hold_time
float32[] progress_joint_id
float32[] progress_joint_angle

